pkg/occlient/occlient_test.go

```diff
@@ -3,12 +3,13 @@ package occlient
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/redhat-developer/odo/pkg/testingutil"
 	"reflect"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/redhat-developer/odo/pkg/testingutil"
+
```

Comment:
why is this here? should it be grouped together wiht 
```
applabels "github.com/redhat-developer/odo/pkg/application/labels"
componentlabels "github.com/redhat-developer/odo/pkg/component/labels"
"github.com/redhat-developer/odo/pkg/util"
```

Reply:
No idea to be honest. Just what `gofmt` does. If I modify it manually it should work :+1: